src/components/Main/Main.js

```diff
@@ -30,15 +31,55 @@ export class MainComponent {
 
   addEventListeners() {
     this.sidebarList.addEventListener("click", this.handleListClick.bind(this));
+    window.addEventListener("popstate", this.handleStatePath.bind(this));
   }
 
   handleListClick(e) {
     e.preventDefault();
-    if (e.target.closest(".main__about-link")) {
-      this.about.mount();
-    } else if (e.target.closest(".main__albums-link")) {
+    const { target } = e;
```

Comment:
you use it once in your block code, so i am not sure it is necessary to destruct the object. why not simply access the appropriate property as is through the dot notation?

Reply:
not sure if it makes code any worse